cookbooks/rust/default.rb

```diff
@@ -0,0 +1,11 @@
+rustc_path = "#{ENV['HOME']}/.cargo/bin/rustc"
```

Comment:
__[staging][rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `rustc_path`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->